build.gradle

```diff
@@ -75,8 +75,6 @@ configurations {
 
 dependencies {
 	shade group: "net.shadowfacts", name: "ShadowMC", version: mc_version + "-" + shadowmc_version, classifier: "deobf"
-
-	shade group: "net.dv8tion", name: "JDA", version: "1.2.2_167", classifier: "withDependencies"
```

Comment:
Why was this removed? You need to change the version part. Without this line, JDA won't be included in the compiled mod.

You also need to rename the new JDA jar to have the `withDependencies` classifier at the end, following standard Maven conventions.

The `javadoc` and `sources` jars should be removed.